spec/adapters/heroku_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "spec_helper"
+
+module Suspenders
+  module Adapters
+    RSpec.describe Heroku do
+      it "sets the heroku remotes" do
+        app_builder = double(app_name: app_name)
+        allow(app_builder).to receive(:append_file)
+
+        Heroku.new(app_builder).set_heroku_remotes
+
+        expect(app_builder).to have_received(:append_file).with(anything(), /heroku join --app #{app_name.dasherize}-production/)
+        expect(app_builder).to have_received(:append_file).with(anything(), /heroku join --app #{app_name.dasherize}-staging/)
```

Comment:
Line is too long. [126/80]<br>Do not use parentheses for method calls with no arguments.